01week/rockPaperScissors.js

```diff
@@ -1,51 +1,61 @@
-'use strict';
+"use strict";
 
-const assert = require('assert');
-const readline = require('readline');
+const assert = require("assert");
+const readline = require("readline");
 const rl = readline.createInterface({
   input: process.stdin,
   output: process.stdout
 });
 
-
-function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+function rockPaperScissors(hand1,hand2) {
+  
```

Comment:
instead of calling toLower() and trim() over and over, you could have made 2 different variables that stored the lowerCase trimmed version of hand1 and hand2 that are passed in.